cli/azd/pkg/commands/pipeline/github_provider_test.go

```diff
@@ -111,4 +113,12 @@ func setupGithubAuthMock(mockContext *mocks.MockContext) {
 	}).RespondFn(func(args exec.RunArgs) (exec.RunResult, error) {
 		return exec.NewRunResult(0, "", ""), nil
 	})
+
```

Comment:
FYI - I made some similar changes in #1975, I think you should just be able to "accept theirs" when you merge in the changes.